test/library/packages/UnitTest/GetTest/Primitive_Get_multiple.chpl

```diff
@@ -0,0 +1,28 @@
+use UnitTest;
+//verifies that generic single argument functions aren't counted
+
+/* A normal function which takes a generic single argument*/
+proc dummy_test(test) throws {
+    //This is a dummy test
+    writeln("a");
+}
+
+/* A test function which takes Test Object as argument*/
+proc test_square(test: borrowed Test) throws {
+    // This is a test function
+    writeln("b");
+}
+/* A test function which takes Test Object as argument*/
+proc test_squared(test: borrowed Test) throws {
+    // This is a test function
+    writeln("c");
+}
+
+
+var test: owned Test;
+param a = __primitive("gather tests", test.borrow());
+writeln(a);
+var c = __primitive("get test by name","test_squared");
+var d = __primitive("get test by index",2);
```

Comment:
I'd like to see a version of this call involving a param variable that is declared earlier.  E.g.
```chapel
param idx = 2; 
var d = __primitive("get test by index", idx);
```

Reply:
It is probably already handled by the param for loop, but just in case . . .

Reply:
Sure. I am adding another test.